src/plcc/load_spec/validate_spec/validate_syntactic_spec/LL1Checker_test.py

```diff
@@ -0,0 +1,93 @@
+from pytest import raises
+from plcc.load_spec.load_rough_spec.parse_dividers import parse_dividers
+from plcc.load_spec.load_rough_spec.parse_lines import Line
+from .LL1Wrapper import LL1Wrapper
+from .LL1Checker import LL1Checker
+from .SpecGrammar import SpecGrammar
+from plcc.load_spec.parse_spec.parse_syntactic_spec.parse_syntactic_spec import parse_syntactic_spec
+from .errors import InvalidParameterError
+
+def test_first_set_terminal():
+    checker = setupChecker(["<exp> ::= VAR"])
+    assert checker.firstSets["exp"] == {"VAR"}
```

Comment:
Tests for LL1Checker should be of the form...

"Here is a grammar that we know is not in LL1, when we run our checker on it, it returns an error."

and 

"Here is a grammar that we know is in LL1, when we run our checker on it, it returns no errors."

And we can try different grammars that might give the checker troubles.

----

This is a test for the construction of firstSets, which should not be part of LL1Checker's observable interface (clients of LL1Checker does not and should not care about firstSets).

I think you need a separate module for FirstSets, and such tests should move to it.

Looking ahead, the same is probably true for FollowSets.